source/includes/language-compatibility-table-php.rst

```diff
@@ -0,0 +1,160 @@
+.. sharedinclude:: dbx/compatibility-table-legend.rst
```

Comment:
S: I dont think this is the right include to use here as it mentions MongoDB compatibility